10-inference.Rmd

```diff
@@ -650,6 +626,7 @@ tail(boot15000_means)
 boot_est_dist <- ggplot(boot15000_means, aes(x = mean)) +
   geom_histogram(fill = "dodgerblue3", color = "lightgrey") +
   xlab("Sample mean price per night ($)")
+boot_est_dist
 ```
 
 Let's compare our bootstrap distribution with the true sampling distribution (taking many samples from the population). 
```

Comment:
When putting two plots together for comparison, their axes should be the same. Here the sampling distribution plot looks like it goes from 90 to 300, while the bootstrap dist goes from 100 to 210. Hard to compare two distributions if their axes are different.

Same comment for all groups of plots throughout this chapter.